st/clitests/awss3api_spec.py

```diff
@@ -69,6 +77,10 @@ def get_aws_cli_object(raw_aws_cli_output):
         cli_obj["prefix"] = common_prefixes
     if (content_keys is not None):
         cli_obj["keys"] = content_keys
+    if (content_keys is not None):
```

Comment:
is this if condition correct? I see 3 if statement with the same condition.

Reply:
Good catch, will fix this!
It worked since had content_keys too present.

Reply:
Done